src/metadata/metacontent_handler.cc

```diff
@@ -60,7 +60,7 @@ fs::path MetacontentHandler::getContentPath(const std::vector<std::string>& name
         } else {
             auto fileNames = std::map<std::string, fs::path>();
             for (const auto& p : fs::directory_iterator(folder))
-                if (p.is_regular_file())
```

Comment:
We should you the error handling variant here also

Reply:
Fixed.